lib/rules/custom-media-pattern/index.js

```diff
@@ -9,7 +9,8 @@ const { isRegExp, isString } = require('../../utils/validateTypes');
 const ruleName = 'custom-media-pattern';
 
 const messages = ruleMessages(ruleName, {
-	expected: (pattern) => `Expected custom media query name to match pattern "${pattern}"`,
+	expected: (name, pattern) =>
+		`Expected custom media query name "${name}" to match pattern "${pattern}"`,
```

Comment:
```suggestion
		`Expected "${name}" to match pattern "${pattern}"`,
```

We now generally favour not repeating the rule name in the message if we quote the actual thing.

e.g.

`Expected "${name}" to match pattern "${pattern}" (custom-media-pattern)`

Did we want to make these changes in this pull request or a follow-up one?

Reply:
That's nice and much simpler!

We now have the following rules with a `... to match pattern` message:

```console
$ git grep 'to match pattern'
lib/rules/comment-pattern/index.js:11:    expected: (pattern) => `Expected comment to match pattern "${pattern}"`,
lib/rules/custom-media-pattern/index.js:13:        `Expected custom media query name "${name}" to match pattern "${pattern}"`,
lib/rules/custom-property-pattern/index.js:17:        `Expected custom property name "${name}" to match pattern "${pattern}"`,
lib/rules/keyframes-name-pattern/index.js:14:        `Expected keyframe name "${keyframeName}" to match pattern "${pattern}"`,
lib/rules/selector-class-pattern/index.js:17:        `Expected class selector "${selector}" to match pattern "${pattern}"`,
lib/rules/selector-id-pattern/index.js:14:        `Expected ID selector "${selector}" to match pattern "${pattern}"`,
lib/rules/selector-nested-pattern/index.js:13:        `Expected nested selector "${selector}" to match pattern "${pattern}"`,
```

In this pull request, I'll add custom message formatting and apply a new message pattern to all other rules than `comment-pattern`.

Reply:
Fixed via 19f313e3f